.eslintrc.js

```diff
@@ -32,23 +32,15 @@ module.exports = {
       },
     ],
     'no-confusing-arrow': [0],
-    'no-console': ['error', {allow: ['debug', 'error']}],
-    'no-unused-vars': ['error', {vars: 'all', args: 'all', ignoreRestSiblings: false}],
+    'no-console': [0],
+    'no-unused-vars': [0],
     'react/jsx-filename-extension': [0],
     'react/jsx-indent': ['error', 2],
-    'react/jsx-indent-props': ['error', 2],
-    'react/jsx-no-bind': [
-      'error',
-      {
-        ignoreDOMComponents: false,
-        ignoreRefs: false,
-        allowArrowFunctions: false,
-        allowFunctions: false,
-        allowBind: false,
-      },
-    ],
+    'react/jsx-indent-props': [0],
+    'react/jsx-no-bind': [0],
```

Comment:
It'll be better to fix the errors instead of disabling eslint rules that advance the performance of the application, the implications of these are dire.